src/reducers/language.js

```diff
@@ -0,0 +1,13 @@
+export default function(state='nb', action) {
+  switch (action.type) {
+    case 'SET_LANGUAGE':
+      return isLanguageValid(action.payload) ? action.payload : null;
```

Comment:
Hvis language ikke er valid, så bør ikke state endres. Ville da returnert `state` i stedet for `null`.